### PR TITLE
Update the 'Setting Up PatternFly' page for patternfly.org

### PR DESCRIPTION
**Pull request by dabeng**
This PR updates and simplifies the "setting Up PatternFly page for patternfly.org.

Notable changes:
- Introduction of CDN based instructions as the quickest path to entry
- Drop the bower instructions
- Overall simplication

To review these changes, please compare the original to the staged improvement
1. http://www.patternfly.org/get-started/setup/
2. https://bleathem.github.io/patternfly-org/get-started/setup/

### CODE REVIEW

**Comment by dabeng** [comment]
@bleathem , I have replaced snapshots with source code.

**Comment by bleathem** [comment]
@dabeng I added a commit to this PR, please review.

**Comment by dabeng** [comment]
Great solution @bleathem

File: source/get-started/setup/index.md [hunk 1]
```diff
@@ -3,186 +3,116 @@ title: Setting Up PatternFly
...
-            <li>Create the directory with the project name<br/>
-              <kbd>
-                mkdir myProject
-              </kbd>
-            </li>
-          </ul>
-        </p>
-      </div>
+    <p>
+      If you want to try out the components provided by PatternFly in a fast way. Just use these <a href="https://cdnjs.com/libraries/patternfly" target="top">PatternFly</a> CDN links.
```
Comment (andresgalante) [inline source/get-started/setup/index.md]: We don't really publish anything on cdnjs, if we want to recommend using a CDN I think we should consider using rawgit CDN or start maintaining a CDN ourselves.

Comment (bleathem) [inline source/get-started/setup/index.md]: We've setup cloudflare to be the CDN for patternfly.  It seems to be working well so far.

File: source/get-started/setup/index.md [hunk 2]
```diff
@@ -3,186 +3,116 @@ title: Setting Up PatternFly
...
+    <p>Now we can includ the necessary assets by directly specifying their path inside the node_modules folder.</p>
+  </div>
+  <div class="col-xs-12 col-sm-12 col-md-12">
+    <pre class="prettyprint">
+      <code class="language-html">{% escape_html %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/patternfly/3.24.0/css/patternfly.min.css">
```
Comment (andresgalante) [inline source/get-started/setup/index.md]: Once a user installs patternfly with npm, they can consume patternfly css from their node_modules folder and stop using the CDN, right?

**Comment by andresgalante** [comment]
I agree with removing bower, good point!

File: source/get-started/setup/index.md (hunk 1 above)
Comment (andresgalante) [inline source/get-started/setup/index.md]: oh! awesome, I didn't knew.

**Pull request merged**